src/preferences/dialog/dlgprefdeck.cpp

```diff
@@ -708,7 +708,7 @@ void DlgPrefDeck::slotNumDecksChanged(double new_count, bool initializing) {
     // The rate range hasn't been read from the config file when this is first called.
     if (!initializing) {
         setRateDirectionForAllDecks(m_rateDirectionControls[0]->get() == kRateDirectionInverted);
-        setRateRangeForAllDecks(m_rateRangeControls[0]->get() * 100);
+        setRateRangeForAllDecks(static_cast<int>(m_rateRangeControls[0]->get() * 100));
```

Comment:
maybe make 100 100.0 to make it more clear?  (Do we want to round after the multiplication or before?)

Reply:
After. If we round before, then the value would always be 0 :D